packages/framework/src/Controller/Admin/ErrorController.php

```diff
@@ -1,188 +0,0 @@
-    public function errorPagePreviewAction(Request $request, int $code): Response
-    {
-        return $this->renderTemplate($code, $request->getRequestFormat());
-    }
-
-    /**
-     * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param \Symfony\Component\ErrorHandler\Exception\FlattenException $exception
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    public function showAction(
-        Request $request,
-        FlattenException $exception,
-    ): Response {
-        if ($this->isUnableToResolveDomainInNotDebug($exception)) {
-            return $this->createUnableToResolveDomainResponse($request);
-        }
-
-        if ($this->debug) {
-            return $this->createDevelopmentResponse($request, $exception);
-        }
-
-        return $this->createProductionResponse($exception->getStatusCode(), $exception->getHeaders());
-    }
-
-    /**
-     * @param int $statusCode
-     * @param array $additionalHeaders
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    protected function createProductionResponse(int $statusCode, array $additionalHeaders = []): Response
-    {
-        $errorPageStatusCode = $this->errorPagesFacade->getErrorPageStatusCodeByStatusCode($statusCode);
-        $errorPageContent = $this->errorPagesFacade->getErrorPageContentByDomainIdAndStatusCode(
-            $this->domain->getId(),
-            $errorPageStatusCode,
-        );
-
-        return new Response($errorPageContent, $errorPageStatusCode, $additionalHeaders);
-    }
-
-    /**
-     * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param \Symfony\Component\ErrorHandler\Exception\FlattenException $exception
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    protected function createDevelopmentResponse(Request $request, FlattenException $exception): Response
-    {
-        $lastThrowable = $this->exceptionListener->getLastThrowable();
-
-        if ($lastThrowable !== null) {
-            return $this->createTracyResponse($lastThrowable);
-        }
-
-        return $this->renderTemplate($exception->getStatusCode(), $request->getRequestFormat());
-    }
-
-    /**
-     * @param \Throwable $throwable
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    protected function createTracyResponse(Throwable $throwable): Response
-    {
-        Debugger::$time = time();
-        $blueScreen = new BlueScreen();
-        $blueScreen->info = [
-            'PHP ' . PHP_VERSION,
-        ];
-
-        ob_start();
-        $blueScreen->render($throwable);
-        $blueScreenHtml = ob_get_clean();
-
-        return new Response($blueScreenHtml);
-    }
-
-    /**
-     * @param \Symfony\Component\HttpFoundation\Request $request
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    protected function createUnableToResolveDomainResponse(Request $request): Response
```

Comment:
With this change, access to the unknown domain will be displayed as a standard 500 error page. 
Also, when a developer has an active acceptance test environment, no hint will be shown anymore. 
Is it ok, @pk16011990?

Reply:
They will get a standard exception with all the info that they need.